tests/Unit/Domain/Test_Orientations.cpp

```diff
@@ -0,0 +1,181 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
```

Comment:
I view copying as a storage or ownership operation.  I have this Thing but I need to give it away twice.  When I added the second class that needed to own the Thing it didn't somehow become important that I could compare Things, because comparisons don't have anything to do with ownership.

For an example of this, I'll argue that the only reason `unique_ptr<EmbeddingMap>` isn't copyable is because the language won't let us define our own copy constructors on standard library classes.  We added `get_clone` instead, which does exactly the same thing.  (Except it's explicit, but implicit copying of EmbeddingMaps wouldn't have been a big worry.)  And yet we recently rather soundly rejected the idea of having an equality operator there.